### DISCUSSION TITLE
[@types/node] crypto.verify should return a boolean, not a Buffer

### DISCUSSION THREAD

**Discussion by juho** [DefinitelyTyped/DefinitelyTyped#59037] [Issues with a @types package] [answered]
Hi, I'm getting a mismatch on the return type for nodejs's `crypto.verify` which in the API docs states that it should return a boolean when there's no callback, but `@types/node/crypto.d.ts` seems to think it should be a Buffer:

```ts
    /**
     * Calculates and returns the signature for `data` using the given private key and
     * algorithm. If `algorithm` is `null` or `undefined`, then the algorithm is
     * dependent upon the key type (especially Ed25519 and Ed448).
     *
     * If `key` is not a [`KeyObject`][], this function behaves as if `key` had been
     * passed to [`crypto.createPublicKey()`][].
     */
    function verify(algorithm: string | null | undefined, data: NodeJS.ArrayBufferView, key: KeyLike | VerifyKeyObjectInput | VerifyPublicKeyInput, signature: NodeJS.ArrayBufferView): Buffer;
```

API docs:

![image](https://user-images.githubusercontent.com/399937/156020185-afd69ea9-19b6-4bbe-b747-41e6651cb416.png)

[a comment by an automated account is not shown]

**Comment by SimonSchick** [comment]
Should be an easy fix, can you send a PR + test?

**Comment by juho** [comment, marked as answer]
Ah, it seems I got served an ancient types file from the depths of some subdirectory locally. The def seems to be correct since a while ago.
